js/modal.js

```diff
@@ -0,0 +1,134 @@
+const projectWrapper = document.querySelector('#works-wrapper');
+const modal = document.querySelector('#modal');
+const closeModal = document.querySelector('#close-modal');
+const modalContent = document.querySelector('#modal-content');
+closeModal.addEventListener('click', () => modal.classList.add('hidden'));
+
+const projects = [
+  {
+    id: 1,
+    name: 'Tonic',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'tonic1.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'multi-post1.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+  {
+    id: 3,
+    name: 'Tonic',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'tonic2.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+  {
+    id: 4,
+    name: 'Multi-Post Stories',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'multi-post2.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+];
```

Comment:
- [ ] You should stick to the design as much as possible like font, color, design, and spacing. There should be paragraphs but yours should be missing and you should stick to design as much as possible. The colors of the buttons are different, please see the Figma design.
### Your design
![his](https://user-images.githubusercontent.com/81727988/210959184-fdcce045-9388-4192-8a7e-15c1cc5a96cc.png)

### Figma design
![finga](https://user-images.githubusercontent.com/81727988/210959207-36431baf-9bda-4ff5-bbac-983b73fa21bb.png)

Reply:
- [ ] Please, as raised by the previous reviewer; I noticed that few your desktop version pop-up window contents are hidden. Kindly address this issue for great user interaction. You may like to reduce the padding of the parent container and the image size as well. Just ensure it's according to the given template. Thanks. 👍 

## Your Design
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text-removebg-preview (1)](https://user-images.githubusercontent.com/103386183/211005808-7b76920d-8630-4380-aba2-fa6c5e5545e9.png)